settings.gradle.kts

```diff
@@ -1,15 +1,23 @@
+val nodeModulesMsdk = File("${rootProject.projectDir}/libs/SalesforceReact/node_modules/react-native/android") // For stand-alone MSDK builds.
+val nodeModulesTemplate = File("${rootProject.projectDir}/../../node_modules/react-native/android") // For template app builds.
+val hasNodeModules = nodeModulesMsdk.exists() || nodeModulesTemplate.exists()
+
 include("libs:SalesforceAnalytics")
 include("libs:SalesforceSDK")
 include("libs:SmartStore")
 include("libs:MobileSync")
 include("libs:SalesforceHybrid")
-include("libs:SalesforceReact")
-include("native:NativeSampleApps:RestExplorer")
-include("native:NativeSampleApps:MobileSyncExplorer")
+if (hasNodeModules) {
+    include("libs:SalesforceReact")
+} else {
+    logger.warn("Skipping `SalesforceReact` module since local node repository is missing.  Has `yarn install` been run in the directory containing `package.json`?")
```

Comment:
@wmathurin - Since we don't have `node_modules` in the case of the hybrid templates, this allows the MSDK workspace to guard for that and build the other libraries.  A warning is printed.

I'd been considering this anyways as a developer experience enhancement.  This lets the build run, plus gives the developer a solid clue that they need to populate the local repositories.  That's better than the default "unable to resolve artifacts" error message, I suspect.

Reply:
I could update the message to refer to running the `install.sh`, since that has the `yarn install` that would have been missed.

Reply:
That will be a big improvement for our side of the build 💫